flexget/plugins/api/series_list.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import copy
+import logging
+from math import ceil
+
+from flask import jsonify
+from flask import request
+from sqlalchemy.orm.exc import NoResultFound
+from flexget import plugin
+from flexget.plugins.filter.series import FilterSeriesBase
+from flexget.api import api, APIResource
+from flexget.plugins.list import series_list as sl
+
+
+SETTINGS_SCHEMA = FilterSeriesBase().settings_schema
+SERIES_ATTRIBUTES = SETTINGS_SCHEMA['properties']
```

Comment:
This does not appear to be used, can you get rid of whole module plugins.filter.series  dependency here?